### ISSUE TITLE
Debian/Ubuntu requirement: python3-venv

### ISSUE THREAD

**Issue by juanMarinero** [pappasam/coc-jedi#25]
TLDR:
- [vim-plug](https://github.com/junegunn/vim-plug) might show as successful a wrong installation [#1806](https://github.com/neoclide/coc.nvim/issues/1806)
- Debian/Ubuntu OS require `python3-venv`. This is NOT enough: `$ apt-get install python-virtualenv` (neither Anaconda, I guess)

My experience. I hope it helps someone. I am newbie, so my advises might be all wrong, if so just delete issue.

After `pip install -U jedi-language-server`,

- I failed with 
`Plug 'pappasam/coc-jedi', { 'do': 'yarn install --frozen-lockfile && yarn build' }`
vim-plug's `:PlugInstall` seems to accept a full of warnings the `engine "coc" appears to be invalid`
but actually, as soon as open a py file I saw the error:
`[coc.nvim] jedi: Error: Command failed: python3 -m venv /.vim/plugged/coc-jedi/.venv && /.vim/plugged/coc-jedi/.venv/bin/pip install -U jedi-language-server==0.23.1`
Posible soltuion [#1806](https://github.com/neoclide/coc.nvim/issues/1806) and install `python3-venv`. Idk, did no try further this way.

- I succeeded with
1st comment the `Plug` line in `~/.vimrc` and `:PlugClean`
Then I ran `:CocInstall coc-jedi` but in py files did error:
`[coc.nvim] jedi: Error: Command failed: python3 -m venv /home/[$USER]/.config/coc/extensions/node_modules/coc-jedi/.venv && /home/[$USER]/.config/coc/extensions/node_modules/coc-jedi/.venv/bin/pip install -U jedi-language-server==0.23.1`
Also, I tried in terminal (outside VIM):
```
$ python3 -m venv $HOME/.config/coc/extensions/node_modules/coc-jedi/.venv
The virtual environment was not created successfully because ensurepip is not
available.  On Debian/Ubuntu systems, you need to install the python3-venv
package using the following command.
    apt-get install python3-venv
```
As suggested, I just did installed **python3-venv** and ran the `:CocInstall` command again. Happy ending!
